src/lib/notifications/notifications.ts

```diff
@@ -87,6 +88,7 @@ export function useNotificationsListener(queryClient: QueryClient) {
     // handle notifications that are received, both in the foreground or background
     // NOTE: currently just here for debug logging
     const sub1 = Notifications.addNotificationReceivedListener(event => {
+      invalidateCachedUnreadPage()
```

Comment:
Not sure it's _necessary_ here but maybe reasonable to always do that? Not dumping the RQ cache.